tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/api/ActivityController.java

```diff
@@ -455,7 +455,7 @@ public Response cancelOn(@HeaderParam(LRA_HTTP_HEADER) String lraId) {
     @GET
     @Path("/cancelOnFamily")
     @Produces(MediaType.APPLICATION_JSON)
-    @LRA(value = LRA.Type.REQUIRED, cancelOnFamily = {Response.Status.Family.CLIENT_ERROR})
+    @LRA(value = LRA.Type.REQUIRED, cancelOnFamily = {Response.Status.Family.CLIENT_ERROR}, terminal = false)
```

Comment:
`terminal` doesn't matter here I think since we are always returning a Response which leads to Cancel

Reply:
+1